### PR TITLE
View appointment command

### PR DESCRIPTION
**Pull request by quzhetao01**
Added the test classes and renamed relevant terms with "addressbook" into "wellnus"

### CODE REVIEW

**Comment by SimWPEric** [comment]
Can re-run check style and fix trailing whitespace style errors.

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -81,42 +81,42 @@ public void setAddressBookFilePath(Path addressBookFilePath) {
...
     @Override
     public void deletePerson(Person target) {
-        addressBook.removePerson(target);
+        wellNus.removePerson(target);
     }
 
     @Override
     public void addPerson(Person person) {
-        addressBook.addPerson(person);
+        wellNus.addPerson(person);
```
Comment (Sheeepen) [inline src/main/java/seedu/address/model/ModelManager.java:106]: thanks for making the effort to change it to wellNus 🐐

**Review by Sheeepen — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/ViewCommand.java
Comment (darrentfy) [inline src/main/java/seedu/address/logic/commands/ViewCommand.java:1]: LGTM, follows format of other commands closely

**Review by darrentfy — APPROVED** [review]
LGTM, great job with the UI for appointments as well

**Pull request merged**